src/types/MenuType.ts

```diff
@@ -0,0 +1,10 @@
+export type MenuType = {
+  id: number;
+  name: string;
+  image: string;
+  discountRate: number;
+  originalPrice: number;
+  discountPrice: number;
+  status: '판매중' | '품절' | '숨김';
+  stock: number;
```

Comment:
```suggestion
  discountPrice: number;
  status: 'SELL' | 'SOLD_OUT' | 'HIDE';
  stock: number;
```

타입은 영문이었으면 좋겠는데, 영어를 형님이 잘하시니 적잘한거 백엔드랑 상의해서 결정해주시죠 ㅎ

Reply:
백엔드랑 맞춰서 IN_STOCK, OUT_OF_STOCK, HIDDEN으로 갈게요!